### DISCUSSION TITLE
INAV Support for Chinese H743 mini FC

### DISCUSSION THREAD

**Discussion by toxstoxs** [iNavFlight/inav-configurator#1969] [General]
Hi.

I kindly allow myself to ask if there is any chance to have a INAV TARGET for this nice FC.

https://i.postimg.cc/4yH5pjDs/426357972-10163650166039966-6484743649400067015-n.jpg
https://i.postimg.cc/4dSP7XFW/426606551-10163650166064966-2749815786230094485-n.jpg

I got a match searching for it and it seems to belong to a TCMMRC brend.
It comes with latest 4.4.3 BF FW and recognized as MINI_H743_HD

I been trying all the H743 targets present on INAV latest Configurator without get one to work on it. (Not even let me connect to the FC).

Would appreciate your opinion if this support is possible.

Regards

**Comment by MrD-RC** [comment]
New hardware must follow this policy. The hardware manufacturer would need to provide a target and samples. Or arrange for an INAV developer to make a target and provide samples for testing. 

https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md
